src/components/spaces/organisms/organelle-wrapper.tsx

```diff
@@ -231,13 +226,44 @@ export class OrganelleWrapper extends BaseComponent<OrganelleWrapperProps, Organ
     const style = {height: MODEL_HEIGHT, width};
     return (
       <div className={"model-wrapper" + (dropperCursor ? " dropper" : "")} style={style}>
-        <div id={this.props.elementName} className="model" onMouseLeave={this.resetHoveredOrganelle}/>
+        <div id={this.props.elementName} className="model" onMouseLeave={this.resetHoveredOrganelle}
+          ref={this.setModelDomRef} />
         {hoverDiv}
         {droppers}
       </div>
     );
   }
 
+  private createNewModel() {
```

Comment:
I was having trouble getting new cells to render before without completely removing the old model first...looks like this is the logic I needed to make that work!